### PR TITLE
[Doc] Update installation and software update instructions

### LINKED ISSUE #343: FSLeyes: Fill axons - different behavior in Mac vs Linux
**Describe the bug**
The Fill axons button works differently between MacOS (Catalina 10.15.2) and Linux (Ubuntu 20.04 LTS in Windows Subsystem for Linux (WSL)).  I use FSLeyes version 0.33.1 with both, installed according to our RTD documentation page and I reloaded the plugin.

In MacOS, the behavior is as intended and gives me this output:
mac_ADS_seg

In Linux, all the image is filled with the axon color even though all myelin are closed or touching the borders:
linux_ADS_seg

**To Reproduce**
1. Download this mask (it contains myelin mask in gray and each axon is marked by a white dot)
mask

2. Open the mask in FSLeyes
3. Click the Fill axons button

I haven't looked at the possible causes for this yet. First, I would like to know if someone can reproduce the same problem or if the problem is with my installation in WSL. @mathieuboudreau I think you have a Mac installation and @Stoyan-I-A a Linux installation, can you give it a try and let me know if you have the same results? Thanks!

### LINKED ISSUE #393: Floodfill test fails with Linux install
### Context

As reported and commented [here](https://github.com/neuropoly/axondeepseg/issues/390#issuecomment-725733513) in #390, the `test_floodfill_axons_returns_expected_arrays` from `test_postprocessing.py` fails when installing ADS + FSLeyes from scratch in my Linux environment (Ubuntu 20.04 in WSL1)
However, Travis tests are passing.

Here is the difference between the 2 installation procedures:
- In our documentation, before installing FSLeyes in Linux, other packages are installed (`C/C++ compilers` and `wxpython`) [here](https://axondeepseg.readthedocs.io/en/latest/documentation.html#linux-tested-on-ubuntu).
- In Travis, those packages are not installed.

### Test 1 - Installation according to documentation:
-  floodfill test is failing
- in `pip list`: `numpy=1.19.4`, and `Pillow=6.2.0`
- in `conda list`: `numpy=1.16.4`, and `pillow=6.2.0`
- Fill Axons button in FSLeyes is not working as expected (fills all the image outside myelin)

### Test 2 - Installation like Travis:
-  floodfill test is passing
- in `pip list`: `numpy=1.19.4`, and `Pillow=8.0.1`
- in `conda list`: `numpy=1.16.4`, and `pillow=6.2.0`
- Fill Axons button in FSLeyes is working as expected
- All other FSLeyes functions of ADS Control Panel are also working as expected.

I don't know why `C/C++ compilers` and `wxpython` were installed but it seems that it is not required and a few of these packages are not available anymore.
<details>
  <summary>Details</summary>

```
E: Unable to locate package libwebkitgtk-dev
E: Unable to locate package libwebkitgtk-3.0-dev
Err:1 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libasound2-dev amd64 1.2.2-2.1ubuntu1
  404  Not Found [IP: 2001:67c:1360:8001::23 80]
Err:2 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libpulse-dev amd64 1:13.99.1-1ubuntu3.5
  404  Not Found [IP: 2001:67c:1360:8001::23 80]
E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/a/alsa-lib/libasound2-dev_1.2.2-2.1ubuntu1_amd64.deb  404  Not Found [IP: 2001:67c:1360:8001::23 80]
E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/pulseaudio/libpulse-dev_13.99.1-1ubuntu3.5_amd64.deb  404  Not Found [IP: 2001:67c:1360:8001::23 80]
```
</details>

As references, here are the diff for pip list and conda list between test 1 and test 2:

<details>
  <summary>diff pip list</summary>

```
64c64
< matplotlib                    3.1.1
---
> matplotlib                    3.3.3
87c87
< Pillow                        6.2.0
---
> Pillow                        8.0.1
```
</details>

<details>
  <summary>diff conda list</summary>

```
23c23
< cairo                     1.14.12              h8948797_3
---
> cairo                     1.16.0            h3fc0475_1005    conda-forge
39,40c39,40
< expat                     2.2.10               he6710b0_2    anaconda
< fontconfig                2.13.0               h9420a91_0    anaconda
---
> expat                     2.2.9                he1b5a44_2    conda-forge
> fontconfig                2.13.1            h7e3eb15_1002    conda-forge
42,43c42,43
< freetype                  2.10.4               h5ab3b9f_0    anaconda
< fribidi                   1.0.10               h7b6447c_0    anaconda
---
> freetype                  2.10.4               h7ca028e_0    conda-forge
> fribidi                   1.0.10               h36c2ea0_0    conda-forge
49,51c49,51
< gettext                   0.19.8.1             h9b4dc7a_1    anaconda
< glib                      2.66.1               h92f7085_0    anaconda
< graphite2                 1.3.14               h23475e2_0    anaconda
---
> gettext                   0.19.8.1             h5e8e0c9_1    conda-forge
> glib                      2.66.1               h92f7085_0
> graphite2                 1.3.13            h58526e2_1001    conda-forge
56c56
< harfbuzz                  2.4.0                hca77d97_1    anaconda
---
> harfbuzz                  2.7.2                hee91db6_0    conda-forge
58c58
< icu                       58.2                 he6710b0_3    anaconda
---
> icu                       67.1                 he1b5a44_0    conda-forge
75c75
< jpeg                      9b                   habf39ab_1    anaconda
---
> jpeg                      9d                   h36c2ea0_0    conda-forge
86a87
> lcms2                     2.11                 hcbb858e_1    conda-forge
97c98,99
< libglu                    9.0.0                hf484d3e_1    anaconda
---
> libglu                    9.0.0             he1b5a44_1001    conda-forge
> libiconv                  1.16                 h516909a_0    conda-forge
101c103
< libpng                    1.6.37               hbc83047_0    anaconda
---
> libpng                    1.6.37               h21135ba_2    conda-forge
106,109c108,113
< libtiff                   4.0.9                he6b73bb_1    conda-forge
< libuuid                   1.0.3                h1bed415_2    anaconda
< libxcb                    1.14                 h7b6447c_0    anaconda
< libxml2                   2.9.10               hb55368b_3    anaconda
---
> libtiff                   4.1.0                h4f3a223_6    conda-forge
> libuuid                   2.32.1            h14c3975_1000    conda-forge
> libwebp-base              1.1.0                h36c2ea0_3    conda-forge
> libxcb                    1.13              h14c3975_1002    conda-forge
> libxml2                   2.9.10               h68273f3_2    conda-forge
> lz4-c                     1.9.2                he1b5a44_3    conda-forge
113c117
< matplotlib-base           3.1.1            py37hfd891ef_0    conda-forge
---
> matplotlib-base           3.3.3            py37h4f6019d_0    conda-forge
136c140
< pango                     1.45.3               hd140c19_0    anaconda
---
> pango                     1.42.4               h7062337_4    conda-forge
138c142
< pcre                      8.44                 he6710b0_0    anaconda
---
> pcre                      8.44                 he1b5a44_0    conda-forge
143c147
< pixman                    0.40.0               h7b6447c_0    anaconda
---
> pixman                    0.38.0            h516909a_1003    conda-forge
149a154
> pthread-stubs             0.4               h14c3975_1001    conda-forge
181c186
< six                       1.15.0                     py_0    anaconda
---
> six                       1.15.0             pyh9f0ad1d_0    conda-forge
216a222,223
> xorg-libice               1.0.10               h516909a_0    conda-forge
> xorg-libsm                1.2.3             h84519dc_1000    conda-forge
218a226
> xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
221a230,231
> xorg-libxrender           0.9.10            h516909a_1002    conda-forge
> xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
227a238
> zstd                      1.4.5                h6597ccf_2    conda-forge
```
</details>

### Test 3 - Installation like travis + re-install ADS:
-  floodfill test is passing
- in `pip list`: `numpy=1.16.4`, and `Pillow=6.2.0`
- in `conda list`: `numpy=1.16.4`, and `pillow=6.2.0`
- Fill Axons button in FSLeyes working as expected

<details>
  <summary>Packages downgraded with the re-install</summary>
 
``` 
Installing collected packages: numpy, scipy, matplotlib, urllib3, idna, requests, pillow, AxonDeepSeg
  Attempting uninstall: numpy
    Found existing installation: numpy 1.19.4
    Uninstalling numpy-1.19.4:
      Successfully uninstalled numpy-1.19.4
  Attempting uninstall: scipy
    Found existing installation: scipy 1.5.3
    Uninstalling scipy-1.5.3:
      Successfully uninstalled scipy-1.5.3
  Attempting uninstall: matplotlib
    Found existing installation: matplotlib 3.3.3
    Uninstalling matplotlib-3.3.3:
      Successfully uninstalled matplotlib-3.3.3
  Attempting uninstall: urllib3
    Found existing installation: urllib3 1.25.11
    Uninstalling urllib3-1.25.11:
      Successfully uninstalled urllib3-1.25.11
  Attempting uninstall: idna
    Found existing installation: idna 2.10
    Uninstalling idna-2.10:
      Successfully uninstalled idna-2.10
  Attempting uninstall: requests
    Found existing installation: requests 2.25.0
    Uninstalling requests-2.25.0:
      Successfully uninstalled requests-2.25.0
  Attempting uninstall: pillow
    Found existing installation: Pillow 8.0.1
    Uninstalling Pillow-8.0.1:
      Successfully uninstalled Pillow-8.0.1
  Attempting uninstall: AxonDeepSeg
    Found existing installation: AxonDeepSeg 3.2.0
    Uninstalling AxonDeepSeg-3.2.0:
      Successfully uninstalled AxonDeepSeg-3.2.0
  Running setup.py develop for AxonDeepSeg
ERROR: After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts.

We recommend you use --use-feature=2020-resolver to test your packages with the new resolver before it becomes the default.

fslpy 3.4.0 requires dataclasses, which is not installed.
Successfully installed AxonDeepSeg idna-2.7 matplotlib-3.1.0 numpy-1.16.4 pillow-6.2.0 requests-2.20.1 scipy-1.3.0 urllib3-1.24.3
```
</details>

@Stoyan-I-A, let me know if you need more information. numpy and Pillow don't seem to be a problem because both Test 2 and 3 are OK.

### PR DESCRIPTION
**Pull request by Stoyan-I-A**
fixes #393 

Possibly fixes #343 as well

Parts of the documentation for the Linux installation have been removed because they are no longer required.

For details, check #393

### CODE REVIEW

**Comment by mariehbourget** [comment]
@Stoyan-I-A, should we also include in this PR the change in the note ([here](https://axondeepseg.readthedocs.io/en/latest/documentation.html#axondeepseg)) to re-install fsleyes after updating ADS?

**Comment by mathieuboudreau** [comment]
> @Stoyan-I-A, should we also include in this PR the change in the note ([here](https://axondeepseg.readthedocs.io/en/latest/documentation.html#axondeepseg)) to re-install fsleyes after updating ADS?

Yes - let's do that here as well.

**Comment by Stoyan-I-A** [comment]
This PR is ready for review.

File: docs/source/documentation.rst [hunk 1]
```diff
@@ -125,6 +125,11 @@ AxonDeepSeg package with the following commands::
         cd axondeepseg
         git pull
         pip install -e .
+
```
Comment (mariehbourget) [inline docs/source/documentation.rst:128]: I would include this in the blue box note to clarify that it is part of the "re-install".

**Comment by Stoyan-I-A** [comment]
I removed the instructions to re-install FSLeyes. The issues caused by those instructions should be addressed in #441. For now, I suggest to merge this PR in case any Linux users want to try the software.

**Review by mariehbourget — APPROVED** [review]

**Comment by mathieuboudreau** [comment]
Closing because issue #452 overrides these changes

**Pull request closed without merging**